integration-tests/__mocks__/server/io-backend.mock.ts

```diff
@@ -0,0 +1,29 @@
+import { once } from "events";
+import { createServer, Server, IncomingMessage } from "http";
+
+export const startServer = async (
+  port: number,
+  mockGetUserSession: jest.Mock
+): Promise<Server> => {
+  console.log("Creating server");
+  const server = createServer((_, response) => {
+    console.log("message", _.url);
```

Comment:
I don't think this is a problem since we are talking about a mock that is logging an url provided just for testing purpose, we can still remove it anyway, what's your take @michaeldisaro ?